src/NuGet.Core/NuGet.Protocol/Model/PackageVulnerabilitySeverity.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Protocol
+{
+    public enum PackageVulnerabilitySeverity
+    {
+        Low = 0,
```

Comment:
Should we also have a value for Unknown, just in case the service doesn't give us the severity for some reason?